Dockerfile

```diff
@@ -17,18 +20,14 @@ RUN apt-get update && apt-get install -y \
   nodejs
 
 # Install libpostal
-RUN apt-get update && apt-get install -y \
-    autoconf automake build-essential curl git libsnappy-dev libtool pkg-config
-
+RUN apt-get update && apt-get install -y curl autoconf automake libtool pkg-config
 RUN git clone https://github.com/openvenues/libpostal
-
 WORKDIR libpostal
-
-RUN ./bootstrap.sh \
-    && mkdir -p /opt/libpostal_data \
-    && ./configure --datadir=/opt/libpostal_data
-
-RUN make && make install && ldconfig
+RUN ./bootstrap.sh
+RUN mkdir -p /opt/libpostal_data
+RUN ./configure --datadir=/opt/libpostal_data
+RUN make -j4
+RUN sudo make install && ldconfig
```

Comment:
ref: how to install libpostal
https://github.com/openvenues/libpostal#installation-maclinux